tests/Fixtures/Integration/set/@PSR12.test-out.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Vendor\Package;
+
+use FooInterfaceA;
+use FooInterfaceB;
+use BarClass as Bar;
+use OtherVendor\OtherPackage\BazClass;
+use function foo;
+use const BAR;
+
+class Foo extends Bar implements FooInterfaceA
+{
+    use FooTrait;
+    use BarTrait;
+
+    public $aaa = 1;
+    public $bbb = 2;
+
+    public function sampleFunction($a, $arg1, $arg2, $arg3, $foo, $b = null)
+    {
+        if ($a === $b) {
+            bar();
+        } elseif ($a    >    $b) {
```

Comment:
there should be one space

Reply:
?

Reply:
Oh, that's an outstanding issue out of scope of this PR. We already discussed this today. :)

Reply:
PSR-12 only insists on exactly one space to the left, but at least one on the right. php-cs-fixer can't do that at the moment.

Reply:
> ... exactly one space to the left, but at least one on the right.

Seems the same logic used for the left and the right parts: https://www.php-fig.org/psr/psr-12/#62-binary-operators